### PR TITLE
SSH terminal client

### PR DESCRIPTION
**Pull request by ehmry**
A client of the Terminal service that connects to a shell on a remote
SSH server.

It is configured with a "host" file found in its root directory with the
following format:
`<host name="sdf-eu.org" port="22" user="root" pass="foo" known="yes"/>`

The port, pass, and known attributes are optional. The client will first
try to authenticate with a keypair found in the root directory with a
fallback to password authentication. The client will automatically
disconnect from hosts that are not found in "/known_hosts", unless the
"known" attribute is set to the negative in the host file.

---

This will probably not work without the libssh version update applied to the main tree.

### CODE REVIEW

**Comment by ehmry** [comment]
@nfeske this is ready for merging.

File: run/ssh_client.run [hunk 1]
```diff
@@ -0,0 +1,179 @@
+#
+# \brief  Test of 'fetchurl
```
Comment (nfeske) [inline run/ssh_client.run]: This is not completely true.

**Comment by nfeske** [comment]
Very cool!

I wonder, what does sdf-eu mean? Wouldn't pkg/ssh_terminal be more intuitive?

Would you mind to move the information given in the commit message to a README file? So it becomes more prominently visible.

**Comment by ehmry** [comment]
The sdf-eu package was a preconfigured client that connected to a public Unix machine, but I've removed that package for now. The test and README should be fixed now.

**Comment by nfeske** [comment]
Thanks. I merged the new commit to master.

**Pull request closed without merging**